geospaas_harvesting/crawlers.py

```diff
@@ -99,17 +99,20 @@ class WebDirectoryCrawler(Crawler):
         f'^.*/{YEAR_PATTERN}/{MONTH_PATTERN}/{DAY_OF_MONTH_PATTERN}/.*$')
     DAY_OF_YEAR_MATCHER = re.compile(f'^.*/{YEAR_PATTERN}/{DAY_OF_YEAR_PATTERN}(/.*)?$')
 
-    def __init__(self, root_url, time_range=(None, None), excludes=None):
+    def __init__(self, root_url, time_range=(None, None), excludes=None, preventer_re=''):
```

Comment:
As suggested by Adrine, replace two input arguments `excludes` and `preventer_re` with just one.